### PR TITLE
Add meta tags for apple (GCI #67)

### PR DESCRIPTION
**Pull request by pythongiant**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#67`
- Preview Link
-https://pythongiant.github.io/gci18.fossasia.org/ 

## Description / Changes
Add Apple Web App Meta

### CODE REVIEW

File: css/custom.css [hunk 1]
```diff
@@ -17,8 +17,16 @@
   text-align: center;
   text-decoration: none;
 }
+.ti-google{
```
Comment (jogendra) [inline css/custom.css]: Work on one issue only in one PR.

Comment (pythongiant) [reply css/custom.css]: umm excuse me?

Comment (jogendra) [reply css/custom.css]: The title referred to adding meta tags for Apple then why are you changing the CSS classes?

Comment (pythongiant) [reply css/custom.css]: oh lemme remove that. 
thanks

File: _includes/header.html [hunk 2]
```diff
@@ -32,6 +32,17 @@
   <meta name="og:site_name" content="Google Code-In with FOSSASIA 2018">
   <meta name="og:video" content="https://gci18.fossasia.org/video/fossasia.mp4">
   <meta name="og:type" content="website">
+
```
Comment (fragm3) [inline _includes/header.html]: Remove unnecessary newlines

File: css/custom.css [hunk 3]
```diff
@@ -17,8 +17,8 @@
   text-align: center;
   text-decoration: none;
 }
+.large .ti-twitter{
```
Comment (fragm3) [inline css/custom.css]: Don't add unnecessary newlines

File: index.html [hunk 4]
```diff
@@ -17,6 +17,7 @@ <h6 class="uppercase mb16" id="new_text">Creating collaborative open-source tech
                         <p class="lead mb40" id="new_text">
                             Connecting Open Tech Communities all over Asia with Developers and Creators
                             <br /> in the FOSSASIA Network.
+
```
Comment (fragm3) [inline index.html]: Remove unnecessary newlines

File: index.html [hunk 5]
```diff
@@ -221,6 +222,7 @@ <h2> Videos </h2>
             </div>
         </div>
         <!--end of row-->
+
```
Comment (fragm3) [inline index.html]: Remove unnecessary newlines

File: index.html [hunk 6]
```diff
@@ -240,6 +242,7 @@ <h5 class="person-name">{{ video.user }}</h6>
                         </div>
                     </a>
                     {% endfor %}
+
```
Comment (fragm3) [inline index.html]: Remove unnecessary newlines

File: index.html (hunk 5 above)
Comment (pythongiant) [reply index.html]: nothing to big

File: index.html (hunk 4 above)
Comment (pythongiant) [reply index.html]: nothing major

**Pull request merged**